atest/test/02_Content_Keywords/files.robot

```diff
@@ -8,17 +8,35 @@ Test Teardown       Run Keywords    Restore Library Timeout    AND    Wait For A
 ${CUSTOM_DL_PATH} =     ${CURDIR}/download_file
 
 *** Test Cases ***
-Upload File
+Upload upload_test_file
+    Upload Named File   test_upload_file
+
+# 75 starts to fail with firefox and chromium
```

Comment:
Do we need these comment lines?

Reply:
Fixed in https://github.com/MarketSquare/robotframework-browser/pull/1511